packages/web/src/components/data-entry/ContextualMenu.module.css

```diff
@@ -0,0 +1,42 @@
+.root {
+  gap: var(--unit-2);
+  padding: var(--unit-4) var(--unit-6);
+  flex-direction: column;
+}
+
+.preview {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--unit-2);
```

Comment:
`<div classNames={cn(layoutStyles.col, layoutStyles.gap2)} />` will do this

Reply:
oh rightt, hell yeah